yksmp/.gitignore

```diff
@@ -0,0 +1,2 @@
+Cargo.lock
+target/
```

Comment:
This file is not necessary anymore.

Reply:
I didn't know that! @ptersilie could be in another/the next PR?

Reply:
Will do!